playwright-e2e/data/tab_content/contested/express_case_gatekeeping_tabs.ts

```diff
@@ -0,0 +1,16 @@
+export const expressCaseGateKeepingEnrolledTabData = [{
```

Comment:
I think @al-hmcts created one in gatekeeping_and_allocation folder.

Reply:
Moved my additional exports into that, and used instead.  Thanks.